src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import styles from './ButtonGroup.module.css';
+
+interface ButtonGroupPropTypes {
+  Buttons: React.ComponentType;
+}
+function ButtonGroup({Buttons} : ButtonGroupPropTypes) {
+  return (
```

Comment:
you could have avoided this extra div, the styles can be passed as props to the button

Reply:
Noted. I think that will reduce the unnessary use of div. Also i used the div because the button group div was used to house a group of buttons . i.e a flex display container to house all flex displayed buttons. Thanks.